### ISSUE TITLE
map ' go

### ISSUE THREAD

**Issue by pevu** [1995eaton/chromium-vim#473]
I'd like to use single- and double quotation marks for quickmarks (has anybody ever used scrollmarks on a webpage?), but I can't get it to work. I've tried to remap `'` to `go` and `openQuickMark`, neither work. Neither does

    map c go

The mappings either do nothing or result in a "mark not set" error. I've tried this in the command prompt and in the options cvimrc textfield with the Chrome Webstore version of cVim. I realize I could `map 'h :open website.tld<cr>`, but that's more of a hack. And `go`/`gn` should be mappable.

In short, please make these work

    map ' go
    map " gn

so `'h` can be used to open the quickmark `h`.
